src/huggingface_hub/inference/_providers/clarifai.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Dict, Optional
+
+from huggingface_hub.hf_api import InferenceProviderMapping
+
+from ._common import BaseConversationalTask
+
+_PROVIDER = "clarifai"
+_BASE_URL = "https://api.clarifai.com"
+
+class ClarifaiConversationalTask(BaseConversationalTask):
+    def __init__(self):
+        super().__init__(provider=_PROVIDER, base_url=_BASE_URL)
+
+    def _prepare_route(self, mapped_model: str, api_key: str) -> str:
+        return "/v2/ext/openai/v1/chat/completions"
```

Comment:
```suggestion
```

Can be removed if the base url is set correctly (the `/v1/chat/completion` part is automatically added)